packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -1008,3 +1012,7 @@ String getNameForHostPlatformArch(HostPlatform platform) {
       return 'x64';
   }
 }
+
+extension on String {
```

Comment:
Don't use extension methods in the Flutter repo: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-extension

This can converted to a helper function instead.